frontend/src/Context/index.tsx

```diff
@@ -21,7 +24,10 @@ const initialState: QuickstartState = {
   linkSuccess: false,
   isItemAccess: true,
   isPaymentInitiation: false,
+  isCraProductsExclusively: false,
+  isUserTokenFlow: false,
```

Comment:
Can we just use `userToken != null` to mean the same thing?

Reply:
see comment below